sections/how-we-impact.liquid

```diff
@@ -0,0 +1,163 @@
+{{ 'section-how-we-impact.css' | asset_url | stylesheet_tag }}
+
+{%- style -%}
+  .how_we_impact_bg_container {
+    margin-top: {{ section.settings.margin_top | times: 0.75 | round: 0 }}px;
+    margin-bottom: {{ section.settings.margin_bottom | times: 0.75 | round: 0 }}px;
+  }
+  @media screen and (min-width: 750px) {
+    .how_we_impact_bg_container {
+      margin-top: {{ section.settings.margin_top }}px;
+      margin-bottom: {{ section.settings.margin_bottom }}px;
+    }
+  }
+{%- endstyle -%}
+
+<div class = "page-width how_we_impact_bg_container">
+    {% if section.settings.how_we_impact_bg_img != blank %}
+        <div class= "how_we_impact_bg_img">
+            {{ section.settings.how_we_impact_bg_img | image_url: width: 430 | image_tag: fetchpriority: fetch_priority }}
+        </div>
+    {% endif %}
+    {% if section.settings.subtitle_icon != blank %}
+        <div class= "how_we_impact_subtitle_icon">
+            {{ section.settings.subtitle_icon | image_url: width: 27 | image_tag: fetchpriority: fetch_priority }}
+        </div>
+    {% endif %}
+    {% if section.settings.subtitle_heading != blank %}
+        <p class="how_we_impact_subtitle_heading">
+          {{ section.settings.subtitle_heading }}
+        </p>
+    {% endif %}
+    {% if section.settings.heading != blank %}
+        <h2
+          class="how_we_impact_heading_text"
+        >
+          {{ section.settings.heading }}
+        </h2>
+    {% endif %}
+    {% if section.settings.text1 != blank %}
+        <div
+          class="how_we_impact_heading_text1"
+        >
+          {{ section.settings.text1 }}
+        </div>
+    {% endif %}
+    {% if section.settings.text2 != blank %}
+        <div
+          class="how_we_impact_heading_text1 how_we_impact_heading_text2"
+        >
+          {{ section.settings.text2 }}
+        </div>
+    {% endif %}
+    {% if section.settings.button_label != blank %}
+        {% if section.settings.button_link != blank %}
+            <a href= "{{ section.settings.button_link }}">
+                <button class= "how-we-impact-button">
+                    {{ section.settings.button_label }}
+                </button>
+            </a>
+        {% else %}
+            <button class= "how-we-impact-button">
+                {{ section.settings.button_label }}
+            </button>          
+        {% endif %}
+    {% endif %}
+    {% if section.settings.rating_text != blank %}
+        <div class= "how_we_impact_subtitle">
+            {% for i in (1..5) %}
+                <span style= "color: var(--special-text)">&#9733;</span>
```

Comment:
instead of adding stars like this, add a snippet here which contains sag of 5 starts.